bolt-sidecar/src/client/commit_boost.rs

```diff
@@ -101,3 +141,41 @@ impl SignerECDSA for CommitBoostSigner {
         Ok(alloy_sig)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use rand::Rng;
+
+    #[tokio::test]
+    async fn test_bls_commit_boost_signer() {
+        let signer =
+            CommitBoostSigner::new("http://localhost:19551".to_string(), "jwt_hex").await.unwrap();
```

Comment:
This will fail in CI, we should use a pattern similar to the others where we read env variables to get the URL, and if not provided, skip the tests.